tinker/e_announcements/templates/ea-future.html

```diff
@@ -129,13 +129,14 @@
         function changeInfo() {
             $.ajax({
                 type: 'POST',
-                url: "/e-announcements/test",
+                url: "/e-announcements/ea_future",
```

Comment:
can we use url_for for this? its a flask method that can be used in jinja (i should have caught that earlier)